components/sync15/src/util.rs

```diff
@@ -15,20 +15,6 @@ use std::{fmt, num};
 #[derive(Debug, Copy, Clone, PartialEq, PartialOrd, Default)]
 pub struct ServerTimestamp(pub i64);
 
-impl From<ServerTimestamp> for i64 {
```

Comment:
these confused me and aren't used, so I think they should die - different semantics for the same value but different types is surely a foot-gun!